docker/Dockerfile

```diff
@@ -61,8 +61,26 @@ COPY bin/ehsm-core/libenclave-ehsm-core.signed.so /home/ehsm/ehsm-core/libenclav
 #    && cd ehsm \
 #    && make
 
```

Comment:
I would suggest COPY the libehsmnapi.so and libenclave-ehsm-core.signed.so in Dockerfile but not commit them into this patch.